1-exercises/B-array-some/exercise.js

```diff
@@ -8,8 +8,16 @@
 
 let pairsByIndex = [[0, 3], [1, 2], [2, 1], null, [3, 0]];
 
+let exit = pairsByIndex.some(k => k==null);
+
+if (exit){
+  return process.exit(1) 
```

Comment:
Great use of process.exit method.  You have obviously read the documentation linked.